### DISCUSSION TITLE
Welcome to Inbucket Discussions!

### DISCUSSION THREAD

**Discussion by jhillyerd** [inbucket/inbucket#192] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Elizaveta1105** [comment]
Hello! I've set up Inbucket on my desktop, but I don't receive emails to the domain that I've specified in config.md.
Does anyone know which additional settings I need to do to receive the emails?
